src/main/java/com/authlete/common/dto/ClientExtension.java

```diff
@@ -54,6 +54,10 @@ public class ClientExtension implements Serializable
     private long refreshTokenDuration;
     private boolean tokenExchangePermitted;
 
+    /**
+     * @since Authlete 3.0.14
+     */
+    private long idTokenDuration;
 
```

Comment:
Could you add another empty line here? (We normally have two empty lines between the last property and the first  method.)